New package: CoordGridTransforms v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CoordGridTransforms
- Repository: https://github.com/JuliaEarth/CoordGridTransforms.jl
- Created by: @eliascarv
- Version: v0.1.0
- Commit: af2610a6598226e7eddfe2f71ed3634426bf8ca0
- Reviewed by: @eliascarv
- Reference: https://github.com/JuliaEarth/CoordGridTransforms.jl/commit/af2610a6598226e7eddfe2f71ed3634426bf8ca0#commitcomment-149936557
- Description: Grid tranforms for CoordRefSystems.jl
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd675243b626a9f29a5f3695fc93ae168b029f362b48cdfb5a2e1aa7e164bc697755208d52eeadc1f87ce19e5051d2b30d6721808f6d0e56bcdb05c89ef42406cc2855371f3490eaa49e530bca9587ab19126986a542ad055f0d1fc8e47efab62491c34cd5c652549b6ba9815a925208cb46bde60a47da7d77fa2e3027b28a4cfd09378a3f2c183d7251c068ca671a082b2787aca5fb1f1d338c723addd7e0e0615764edd38b788d645d7a6bc2950940b65e4759e2d9d9eeb624cd19e416cb1dafc -->

goerz commented:
Could you add just a little bit more documentation in the README to explain how exactly this can be used? Maybe it's extremely obvious to users of [CoordRefSystems](https://github.com/JuliaEarth/CoordRefSystems.jl), but adding a paragraph of explanation of _how_ `CoordGridTransforms` acts as an "extension" still seems like a good idea

[noblock]

JuliaRegistrator committed:
New package: CoordGridTransforms v0.1.0

UUID: 1cf1ec0c-22f7-4062-925b-8db09db48fc3
Repo: https://github.com/JuliaEarth/CoordGridTransforms.jl.git
Tree: 527bb5a8201480dfc543b1582f709a4e915fa1d8

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

eliascarv commented:
@goerz done!

[noblock]

eliascarv commented:
@goerz why is the registration blocked? Any other pending issues?

goerz commented:
[noblock] What makes you say that it's blocked?

[As the bot says](https://github.com/JuliaRegistries/General/pull/120627#issuecomment-2515533496):

> Your new package registration met all of the guidelines for auto-merging and is scheduled to be merged when the mandatory waiting period (3 days) has elapsed.

That waiting period elapses in a few hours, so it will be merged then

eliascarv commented:
Oh sorry, I got confused when I saw "Merging is blocked" in the PR.

![Captura de tela de 2024-12-06 15-49-03](https://github.com/user-attachments/assets/34717ec4-6efe-488c-9027-fd607e5fdf1c)

[noblock]

goerz commented:
[noblock] That just means you do not have the permissions to merge this manually (nor do I)
